app/services/idv/session.rb

```diff
@@ -3,6 +3,7 @@ class Session
     VALID_SESSION_ATTRIBUTES = %i[
       address_verification_mechanism
       applicant
+      document_expired
```

Comment:
it ends up hurting to add `reproof_at`, because the constants are used to define setters as well as getters via `method_missing`, and then an automated spec runs through and has issues with the setter, so it was easier to define just the one `document_expired`